src/dispatch/experimental/durable/frame.c

```diff
@@ -138,24 +139,12 @@ static PyGenObject *get_generator_like_object(PyObject *obj) {
     return NULL;
 }
 
-static InterpreterFrame *get_interpreter_frame(PyObject *obj) {
-    PyGenObject *gen_like = get_generator_like_object(obj);
-    if (!gen_like) {
-        return NULL;
-    }
+static InterpreterFrame *get_interpreter_frame(PyGenObject *gen_like) {
```

Comment:
Was there a reason why you chose to name this type `IterpreterFrame` instead of `_PyInterpreterFrame` like it's named in the interpreter code?

Reply:
I think I chose the name while experimenting with earlier Pythons, before they had the `_frame` / `FrameObject` => `_InterpreterFrame` split (which came in Python 3.11).